frontend-project/src/services/institutions.ts

```diff
@@ -1,5 +1,30 @@
 import smallEodSDK from '@/utils/sdk';
 
+export interface Institution {
+  name: string;
+  administrativeUnit: string;
+  modifiedBy: number;
+  createdBy: number;
+  modifiedOn: string;
+  createdOn: string;
+  email: string;
+  city: string;
+  epuap: string;
+  street: string;
+  houseNo: string;
+  postalCode: string;
+  flatNo: string;
+  nip: string;
+  regon: string;
+}
+
+export const fetchInstitution = async (id: number): Promise<Institution> => {
+  smallEodSDK.InstitutionsApi();
```

Comment:
Jaką to pełni funkcje?

Reply:
Myślę, że @mateuszmagulski odtwarza dosyć bezmyślnie to co ja napisałem bezmyślnie tutaj: https://github.com/watchdogpolska/small_eod/pull/259/files#diff-b39b02d634388c4ea3eeb23c3662f3b7R22

A wszystko opiera się na: https://github.com/watchdogpolska/small-eod-sdk-javascript/blame/master/README.md#L102 (gdzie nie zainicjowanie objektu `api` powoduje że `institutionsRead` na `smallEodSDK` jest `undefined`).

Reply:
To może prościej ``` smallEodSDK.InstitutionsApi().institutionsRead(id)``` zgodnie z przykładami w dokumentacji https://github.com/watchdogpolska/small-eod-sdk-javascript/blob/master/docs/InstitutionsApi.md#example-4 ?

Reply:
Nawet podlinkowany przez Ciebie przykład robi tak jak pisałem, bo pobiera ```institutionsRead``` z ```CasesApi```, zamiast z ```SmallEodClient```.